app/Http/Controllers/PermissionGroup/PermissionGroupController.php

```diff
@@ -58,7 +68,11 @@ public function show($id)
      */
     public function edit($id)
     {
-        //
+        $permissionGroup = $this->permissionGroupRepository->findById($id);
```

Comment:
$permissionGroup = $this->permissionGroupRepository->findById($id);

=>

if (! $permissionGroup = $this->permissionGroupRepository->findById($id)) {
abort(404);
}